Tribler/community/metadata/__init__.py

```diff
@@ -0,0 +1,3 @@
+'''
+The metadata package contains TTTODO: what does this do?
```

Comment:
The metadata package contains a Dispersy community which can be used to disseminate metadata of torrents